tests/integration/test_api_gateway.py

```diff
@@ -842,6 +842,59 @@ def test_base_path_mapping(self):
         with self.assertRaises(Exception):
             client.delete_base_path_mapping(domainName=domain_name, basePath=base_path)
 
+    def test_base_path_mapping_root(self):
```

Comment:
Thanks for fixing this @shawnsilva ! 👍 

nitpick: looks like this method is very similar to `test_base_path_mapping(..)` above - we could extract the common parts and then parameterize the test. 

(Not critical, though - we'll likely refactor this test to `pytest` at some point, and can then achieve this via `@pytest.mark.parametrize`).